benchmarks/smallmatvec/manual_versions/libxsmm/Makefile

```diff
@@ -0,0 +1,63 @@
+FC=ifort
+FFLAGS=-qopenmp -Ofast
+LDFLAGS=
+
+# MKLFLAGS="-L${MKLROOT}/lib/intel64 -mkl"
+XSMM_INCLUDE=-I/lustre/scafellpike/local/HCP055/pxs02/rxf70-pxs02/libxsmm-1.15/include
```

Comment:
Probably best to remove your user details from this. I suggest maybe:

    XSMM_DIR ?= ${HOME}/libxsmm-1.15
    XSMM_INCLUDE ?= ${XSMM_DIR}/include
    XSMM_LINK ?= ${XSMM_DIR}/lib -lxsmmf ....

That way it can be overridden by the user.

Reply:
Much nicer thanks :-)